src/azure-cli/azure/cli/command_modules/databoxedge/generated/_params.py

```diff
@@ -0,0 +1,253 @@
+        c.argument('sku', action=AddSku, nargs='+', help='The SKU type.')
+        c.argument('etag', type=str, help='The etag for the devices.')
+        c.argument('data_box_edge_device_status', options_list=['--status'], arg_type=get_enum_type(['ReadyToSetup',
+                                                                                                     'Online',
+                                                                                                     'Offline',
+                                                                                                     'NeedsAttention',
+                                                                                                     'Disconnected',
+                                                                                                     'PartiallyDisconnected',
+                                                                                                     'Maintenance']),
+                   help='The status of the Data Box Edge/Gateway device.')
+        c.argument('description', type=str, help='The Description of the Data Box Edge/Gateway device.')
+        c.argument('model_description', type=str, help='The description of the Data Box Edge/Gateway device model.')
+        c.argument('friendly_name', type=str, help='The Data Box Edge/Gateway device name.')
+
+    with self.argument_context('databoxedge device update') as c:
+        c.argument('device_name', options_list=['--name', '-n', '--device-name'], type=str, help='The device name.',
+                   id_part='name')
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('tags', tags_type)
+
+    with self.argument_context('databoxedge device delete') as c:
+        c.argument('device_name', options_list=['--name', '-n', '--device-name'], type=str, help='The device name.',
+                   id_part='name')
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('databoxedge device download-update') as c:
+        c.argument('device_name', options_list=['--name', '-n', '--device-name'], type=str, help='The device name.',
+                   id_part='name')
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('databoxedge device install-update') as c:
+        c.argument('device_name', options_list=['--name', '-n', '--device-name'], type=str, help='The device name.',
+                   id_part='name')
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('databoxedge device scan-for-update') as c:
+        c.argument('device_name', options_list=['--name', '-n', '--device-name'], type=str, help='The device name.',
+                   id_part='name')
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('databoxedge device show-update-summary') as c:
```

Comment:
so many update related commands. could we group them?

Reply:
I checked powershell command structure, they didn't add a group for them. And I didn't come up with a proper group name so...😂